canonical-web-and-design/maas-ui
# fix(ui): update dashboard file organisation + config form

## Done

- Updated the dashboard files so that only the content below the header is routed instead of each tab being entirely different components
- Added tests + fixed some minor bugs with the dashboard config form

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to /MAAS/r/dashboard/configuration and check that the values save correctly
- Refresh the page and check that there are no console errors

## Screenshot

![2021-06-18_16-38](https://user-images.githubusercontent.com/25733845/122517698-b3108780-d053-11eb-9aff-790a3ca15688.png)
